backend/tests/Unit/Service/MetadataAccessResolverTest.php

```
@@ -0,0 +1,332 @@
+    /** With nothing shared to fall back to, it reads as unconfigured. */
+    public function testAnIgnoredPersonalTokenLeavesNothingToFallBackOn(): void
+    {
+        $resolver = $this->resolver(
+            configuration: static function (MetadataProviderConfiguration $c): void {
+                $c->setPersonalCredentialsEnabled(false);
+            },
+            metronSharedAllowedByEnvironment: true,
+            personal: (new UserMetadataCredential())->setMetronToken('personal-token')
+        );
+
+        self::assertFalse($resolver->resolve('metron', $this->user())->isGranted());
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Unit\Service\MetadataAccessResolverTest::assertFalse().


(staticMethod.notFound)